src/main/java/com/ldtteam/blockout/Pane.java

```diff
@@ -525,6 +525,9 @@ public void putInside(final View newParent)
         if (parent != null)
         {
             parent.addChild(this);
+
+            // Allow views to expand zero-widths
+            setSize(width, height);
```

Comment:
if this instanceof View...

Reply:
Due to polymorphism this doesn't really matter much. Though arguably that may have been more maintainable.